clang/lib/Sema/SemaARM.cpp

```diff
@@ -810,7 +836,7 @@ bool SemaARM::CheckNeonBuiltinFunctionCall(const TargetInfo &TI,
 #undef GET_NEON_IMMEDIATE_CHECK
   }
 
-  return SemaRef.BuiltinConstantArgRange(TheCall, i, l, u + l);
+  return ParseNeonImmChecks(TheCall, ImmChecks, TV);
```

Comment:
I think using TV here is unnecessary. If we always specified the index of the base type for each intrinsics this could be removed and we could merge ParseNeon and ParseSVE